src/io/g15.rs

```diff
@@ -0,0 +1,251 @@
+#![allow(non_upper_case_globals)]
+#![allow(unused_variables)]
+
+use regex::{Regex, Captures};
+use std::fmt;
+use std::fs;
+use std::num::ParseIntError;
+
+#[derive(Debug)]
+pub enum G15BadIndex {

```

Comment:
Errors should have a more generic name, it's not just for bad indices right? It's also for when an int fails to parse. Because modules namespace the types it can just be something generic like `Error`

Reply:
oops artifact of the ages. that is meant to be G15Error yes, that is the name from when it was a struct